narwhals/stable/v1/__init__.py

```diff
@@ -1496,7 +1496,7 @@ def when(*predicates: IntoExpr | Iterable[IntoExpr]) -> When:
     `.otherwise(<value if condition is false>)` can be appended at the end. If not
     appended, and the condition is not `True`, `None` will be returned.
 
-    !!! info
+    Info:
 
```

Comment:
Fixes one of (#2481), but won't show in API reference

```suggestion
    Info:
```

Currently looks like a code block 😞 

![image](https://github.com/user-attachments/assets/6faa0651-1f43-476d-b016-3446ddfa10e6)